src/itest/kotlin/nl/lifely/zac/itest/NotificationsTest.kt

```diff
@@ -150,11 +155,72 @@ class NotificationsTest : BehaviorSpec({
                     Wait.forLogMessage(
                         ".* Er is iets fout gegaan in de Zaaktype-handler bij het afhandelen van notificatie: " +
                             "null ZAAKTYPE CREATE .*: java.lang.RuntimeException: URI 'http://example.com/dummyResourceUrl' does not " +
-                            "start with value for environment variable 'ZGW_API_CLIENT_MP_REST_URL': 'http://openzaak.local:8000/' .*",
+                            "start with value for environment variable 'ZGW_API_CLIENT_MP_REST_URL': '$OPEN_ZAAK_BASE_URI/' .*",
                         1
-                    ).withStartupTimeout(DURATION_THIRTY_SECONDS)
+                    ).withStartupTimeout(THIRTY_SECONDS)
                 )
             }
         }
     }
+    Given("A websocket subscription is created to listen to all changes made to a specific zaak") {
+        val websocketListener = WebSocketTestListener(
+            textToBeSentOnOpen = "{" +
+                "\"subscriptionType\":\"CREATE\"," +
+                "\"event\":{" +
+                "  \"opcode\":\"ANY\"," +
+                "  \"objectType\":\"ZAAK\"," +
+                "  \"objectId\":{" +
+                "    \"resource\":\"$zaak1UUID\"" +
+                "  }," +
+                "\"_key\":\"ANY;ZAAK;$zaak1UUID\"" +
+                "}" +
+                "}"
+        )
+        itestHttpClient.connectNewWebSocket(
+            url = ItestConfiguration.ZAC_WEBSOCKET_BASE_URI,
+            webSocketListener = websocketListener
+        )
+        // we need to wait a bit to make sure the websocket subscription has been registered in ZAC
+        Thread.sleep(FIVE_HUNDRED_MILLISECONDS)
```

Comment:
I know this is far from ideal but I currently don't know how else to solve this. As far as I can see there is nothing I can wait for so to speak. If we want that I think we need to add another endpoint and even a service just to be able to retrieve the current list of websocket subscriptions..

Reply:
Can we use [eventually](https://kotest.io/docs/assertions/eventually.html) for the test?

Same logic (wait for X seconds), but better semantics I think

Reply:
Well, not quite the same I think because eventually (like awaitility) always needs to wait for some condition to pass and I have no such condition to check here.. Except of course sending that notification and checking that that event is sent to the websocket. Maybe I should do that indeed..

Reply:
I fixed it now and also switched to using `eventually` in other places where we were using `awaitlity`. `eventually` is much better indeed. Thanks for pointing me to that!